server/controller/user-contoller.js

```diff
@@ -21,21 +22,22 @@ const signUp = async(req,res,next) =>{
    const { name , email , password } = req.body;
 
    let existingUser;
-
    try{
     existingUser = await User.findOne({email})
    }catch(e){
     console.log(err);
```

Comment:
The error object 'err' should not be directly logged as it may contain sensitive information. Consider logging a generic error message or sanitizing the error object before logging.

```suggestion
console.error("An error occurred");
```